ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbar.java

```diff
@@ -290,6 +290,7 @@ private void setIconSource(ReadableMap source, IconControllerListener controller
               .setOldController(holder.getController())
               .build();
       holder.setController(controller);
+      holder.getTopLevelDrawable().setVisible(true, true);
```

Comment:
>> Can getTopLevelDrawable return null?
No, it can't. Root drawable in DraweeHolder exists all time.

>>What is true, true?
First `true` turn on visibility. There is no effect If drawable is already visible. Second `true` resets drawable animation to start. Although animations is not enabled now it can be turned on in future (with one line). Param has no any effect for now.

>> Ah so uri being null in a previous call makes this invisible?
System calls setVisible() of Drawable with `false` if you set logo or icon to `null`. And then if you set `uri` to valid path then Drawee will not trigger image fetch request.